CHANGES.md

```diff
@@ -40,6 +50,7 @@
 
 **Merged pull requests:**
 
+- library: update resource for acquisition settings [\#2361](https://github.com/rero/rero-ils/pull/2361) ([zannkukai](https://github.com/zannkukai))
```

Comment:
This should not be merged on `dev`. Only on the `US_acquisition` branch.
Should be removed